packages/data-explorer/src/types/RouteQuery.ts

```diff
@@ -3,5 +3,6 @@ export type RouteQuery = {
     size?: string, // page size integer as a string
     sort?: string, // name of sort column, prefix with '-' for reverse sort
     filter?: string // hashed filter object
+    filterbar?: '0' | '1' // filter sidebar
```

Comment:
why is an int (as string) instead of a bool ( as string) , why filterbar and not 'showfilters' or 'isfilersshown' or something like that

Reply:
I suppose its a matter a of personal preference. I find an int easier to parse to a boolean and to toggle states between. Same for the naming of the variable; I would rather not use verbs, because the other parameters are nouns.